test/vfd_swmr_gperf_writer.c

```diff
@@ -0,0 +1,2979 @@
+                else {
+                    printf("Invalid -A argument \"%s\"", optarg);
+                    TEST_ERROR;
+                }
+                break;
+            case 'q':
+                verbosity = 0;
+                break;
+            case '?':
+            default:
+                usage(s->progname);
+                break;
+        }
+    }
+    argc -= optind;
+    argv += optind;
+
+    if (s->grp_op_pattern != ' ')
+        s->grp_op_test = true;
+    if (s->at_pattern != ' ')
+        s->attr_test = true;
+
+    if (!s->grp_op_test) {
+        if (s->asteps < 1 || s->asteps > s->nsteps) {
+            printf("attribute interval is out of bounds\n");
+            TEST_ERROR;
+        }
+    }
+
+    if (s->grp_op_test && s->attr_test) {
+        printf("Cannot test both group operation and attribute tests!\n");
+        printf("Attribute tests are ignored.\n");
+    }
+
+    if (argc > 0) {
+        printf("unexpected command-line arguments\n");
+        TEST_ERROR;
+    }
+
+    /* space for attributes */
+    if ((s->one_by_one_sid = H5Screate_simple(1, &dims, &dims)) < 0) {
+        printf("H5Screate_simple failed\n");
+        TEST_ERROR;
+    }
+
+    esnprintf(s->filename, sizeof(s->filename), "vfd_swmr_group.h5");
+
+    return true;
+
+error:
+    if (tfile)
+        HDfree(tfile);
+    return false;
+}
+
+/*-------------------------------------------------------------------------
+ * Function:    check_ohr_num_chunk
+ *
+ * Purpose:     Check if the number of object header chunks is as expected.
+ *
+ * Parameters:  hid_t oid
+ *              HDF5 object ID (in this file: means group ID)
+ *
+ *              bool one_chunk_ohr
+ *              flag to indicate if the object header chunk is 1 or greater
+ *              1: true
+ *              greater than 1: false
+ *
+ * Return:      Success:    true
+ *              Failure:    false
+ *
+ *-------------------------------------------------------------------------
+ */
+
+static bool
+check_ohr_num_chunk(hid_t g, bool one_chunk_ohr)
+{
+
+    H5O_native_info_t ninfo;
+
+    /* Get the object information */
+    if (H5Oget_native_info(g, &ninfo, H5O_NATIVE_INFO_HDR) < 0) {
+        printf("H5Oget_native_info failed\n");
+        TEST_ERROR;
+    }
+
+    if (true == one_chunk_ohr) {
+        if (ninfo.hdr.nchunks != 1) {
+            printf("Object header should have only one chunk,but it is not.\n");
+            TEST_ERROR;
+        }
+    }
+    else {
+        if (ninfo.hdr.nchunks <= 1) {
+            printf("Object header should have more than one chunk,but it is not.\n");
+            TEST_ERROR;
+        }
+    }
+
+    return true;
+
+error:
+    return false;
+}
+
+/*-------------------------------------------------------------------------
+ * Function:    add_attr
+ *
+ * Purpose:     Add attributes to a group.
+ *
+ * Parameters:  state_t *s
+ *              The struct that stores information of HDF5 file, named pipe
```

Comment:
Remove "named pipe".  There are other places in the file that mentioned "named pipe".